src/app-layout/visual-refresh-toolbar/state/use-app-layout.tsx

```diff
@@ -344,12 +453,16 @@ export const useAppLayout = (
 
   const splitPanelInternals: SplitPanelProviderProps = {
     bottomOffset: 0,
-    getMaxHeight: useStableCallback(() => {
+    getMaxHeight: useCallback(() => {
```

Comment:
Why did this change? The idea of `useStableCallback` is to avoid extra re-renders when a dependency changes, because the function is called lazily